metaflow/plugins/kfp/kfp.py

```diff
@@ -1276,6 +1292,7 @@ def _create_metaflow_step_op(
         step_variables: StepVariables,
         flow_variables: FlowVariables,
         metaflow_configs: Dict[str, str],
+        flow_parameters: Optional[Dict],
         flow_parameters_json: str,
```

Comment:
Can `flow_parameters_json` be cleared out across the board given you already have `flow_parameters`? There are a few other occurrences of `flow_parameters_json` as well.

Reply:
`flow_parameters_json` is still needed throughout the codebase to represent the Argo flow parameters json.
Except that now they are of json form `[{"name": "foo", "value": "bar"}, ...]`